sourmash/sbt_storage.py

```diff
@@ -97,6 +107,137 @@ def __exit__(self, type, value, traceback):
         self.tarfile.close()
 
 
+class ZipStorage(Storage):
+
+    def __init__(self, path=None):
+        # TODO: leave it open, or close/open every time?
+
+        if path is None:
+            # TODO: Open a temporary file?
```

Comment:
do we want to flag things here (raise an exception, or some such)? or just require path in the constructor?

Reply:
I think it would be interesting to do what is happening with the buffer now, where it is kept in memory, and throw a `Warning` if the file is not open? But probably just make it required is easier.